app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -438,10 +438,14 @@ function AcuantCapture(
     const { image, cardtype, dpi, moire, glare, sharpness } = nextCapture;
     const isAssessedAsGlare = glare < glareThreshold;
     const isAssessedAsBlurry = sharpness < sharpnessThreshold;
+    const isUnsupportedDocument = cardtype !== 1;
```

Comment:
Very minor, but: The other variables are named as `isAssessedAsX` where `X` describes the assessment result. This behaves the same way, but is inconsistent in how the variable is named.

I might expect...

```suggestion
    const isAssessedAsUnsupported = cardtype !== 1;
```